packages/reactive-element/src/test/std-decorators/property_test.ts

```diff
@@ -415,4 +415,23 @@ suite('@property', () => {
     await el.updateComplete;
     assert.isTrue(changedProperties!.has('foo'));
   });
+
+  test('decorator on subclass should not break superclass decorators', () => {
+    class BaseTest extends ReactiveElement {
+      @property() accessor first = 'first';
+    }
+
+    class SubClass extends BaseTest {
+      // The presence of this decorator should not cause `first` to break.
+      @property() accessor second = 'second';
+    }
+
+    const elName = generateElementName();
+    customElements.define(elName, SubClass);
+
+    container.innerHTML = `<${elName} first="overrideFirst"></${elName}>`;
+
+    const el: SubClass = container.querySelector(elName)!;
+    assert.equal(el.first, 'overrideFirst');
```

Comment:
I love the practice of adding a test, then fixing that test in a PR, makes it so nice to verify behavior!

For future changes, it'd be better to demonstrate in the first commit the problematic behavior. I think this would involve doing:

```ts
    container.innerHTML = `<${elName} first="overrideFirst" second="overrideSecond"></${elName}>`;

    const el: SubClass = container.querySelector(elName)!;
    assert.equal(el.first, 'overrideFirst');
    // bug! this should be "overrideSecond"
    assert.equal(el.second, 'second');
```

Reply:
Rebased commits so that it follows this excellent pattern.